tests/unit/api/test_simple.py

```diff
@@ -18,10 +18,11 @@
 from pyramid.testing import DummyRequest
 
 from warehouse.api import simple
-from warehouse.packaging.utils import API_VERSION
+from warehouse.packaging.utils import API_VERSION, _valid_simple_detail_context
```

Comment:
I haven't changed the `API_VERSION` to `1.2`, as I wasn't sure if or how PyPI presents different versions of APIs?

Reply:
There's only ever one major version of the API active at once, and the only one we serve is the latest version. It should be fine to update the `API_VERSION` to 1.2, it'll just change the string in the response body.

Reply:
addressed in 6d4a48472bda2b2b240ff8e336086379f2088a7f